src/openzaak/conf/includes/base.py

```diff
@@ -590,14 +379,20 @@
 #
 # DJANGO-CSP
 #
-CSP_CONNECT_SRC = CSP_DEFAULT_SRC + ["raw.githubusercontent.com"]
-CSP_SCRIPT_SRC = CSP_SCRIPT_SRC + ["cdnjs.cloudflare.com", "cdn.jsdelivr.net"]
-CSP_IMG_SRC = CSP_IMG_SRC + ["cdnjs.cloudflare.com", "tile.openstreetmap.org"]
-CSP_STYLE_SRC = CSP_STYLE_SRC + ["cdnjs.cloudflare.com", "cdn.jsdelivr.net"]
 
-# TODO is there a better way to fix this?
-CSP_INCLUDE_NONCE_IN.remove("script-src")  # error with GISModelAdmin.
-CSP_INCLUDE_NONCE_IN.remove("style-src")  # error with redoc.
+CONTENT_SECURITY_POLICY = {
```

Comment:
When I'm running in docker or runserver, the redoc page seems to be broken (e.g. http://localhost:8000/zaken/api/v1/schema/):

<img width="1687" height="126" alt="image" src="https://github.com/user-attachments/assets/4ba864f9-c3ea-4781-b8f9-be0936877e98" />

Could you fix this?